test/sizes.gzip.json

```diff
@@ -0,0 +1,43 @@
+{
+  "amp.js": "92.33 kB",
```

Comment:
Its really sad to loose version history. Can't we read in the old format just as easily using a CSV reader?